tests/system/action/topic/test_delete.py

```diff
@@ -67,3 +67,11 @@ def test_create_delete(self) -> None:
         self.assert_status_code(response, 200)
         self.assert_model_deleted("topic/1")
         self.assert_model_deleted("list_of_speakers/1")
+
+    def test_example_data(self) -> None:
```

Comment:
Maybe also remove or change this test because load_example_data is a really slow test case.

Reply:
Changed.